code/modules/shuttle/special.dm

```diff
@@ -52,6 +52,9 @@
 /obj/machinery/power/emitter/energycannon/magical/emag_act(mob/user)
 	return
 
+/obj/machinery/power/emitter/energycannon/magical/RefreshParts()//monkestation edit, makes it correctly not require power
+	return ..()
```

Comment:
I would have preferred the bugfix being it's own PR, rather than tacked onto a content update.

Reply:
yeah, it was only included because it was a side effect of now scrapped content for this PR, but I can remove it from this